sktime/datatypes/_panel/_examples.py

```diff
@@ -77,6 +77,7 @@
 X["var_1"] = pd.Series(
     [pd.Series([4, 5, 6]), pd.Series([4, 55, 6]), pd.Series([42, 5, 6])]
 )
+X.index.names = ["instances"]
```

Comment:
I have multiple reservations here:

* we are removing coverage from the "no index" case unrelated to the polars mtype. This is not sth we should do, similar to the other discussion.
* why would the conversion to polars fail if the `pd-multiindex` does not have index names? This is a perfectly valid instance of `pd-multiindex` formatted data, so the conversion should not fail in the case. Removing coverage from a case that should work is not the solution imo. Unless of course I misunderstand the situation and you contest that it should work in this case.

Reply:
we are not removing coverage from "no index" case. If no index is identified we don't add any column and convert the dataframe as it is. Regarding multiindex, I will try to explain the failure once again. So the polars fixture comes from directly converting the pd-multiindex fixture to polars, which has index columns named `__index__instances` and `__index__timepoints`. Now when the nested_univ is converted to polars_panel the index name being `None` in that case the index column names turn into `__index__0` and `__index__timepoints` causing difference in the the polars fixture and the converted dataframe, which causes failure during `deep_equals`.

Reply:
are you saying, this is a mistake with the example? If so, it should be a separate PR.

Though, I still wonder whether it really is. When converting to `pd-multiindex`, I think the index names should be simply `None` - both - because the original indices have no names, in the example. Do the index names get added when converting `nested_univ` to `pd-multiindex`?